c7n/resources/mq.py

```diff
@@ -82,6 +83,57 @@ def get_dimensions(self, resource):
         return [{'Name': self.model.dimension,
                  'Value': "{}-1".format(resource['BrokerName'])}]
 
+@MessageBroker.filter_registry.register('vpc')
+class VpcFilter(VpcFilter):
+
+    sg_vpc_dict = {}
+
+    def get_related_ids(self, resources):
+        client = local_session(self.manager.session_factory).client('ec2')
+        related_ids = []
+        for r in resources:
+            sg_ids = r.get('SecurityGroups', [])
+            vpc_id = self.sg_vpc_dict.get(sg_ids[0])
+            if(vpc_id):
```

Comment:
Python doesn't require this to be in parenthesis. This lets us simplify a statement like this to: `if vpc_id:`